src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/CustomConfigTests.kt

```diff
@@ -37,17 +39,10 @@ abstract class CustomConfigTests
     @Before
     fun createDatabase()
     {
-        println("Copying database from: ${AppConfig()["db.template"]}")
-
-        val newDb = AppConfig()["db.location"]
-        val source = AppConfig()["db.template"]
```

Comment:
realised this templating logic was superfluous since we started zipping up the whole orderly demo dir and unzipping a fresh copy before each test